includes/gateways/stripe/includes/give-stripe-helpers.php

```diff
@@ -930,7 +931,7 @@ function give_stripe_process_payment( $donation_data, $stripe_gateway ) {
 					'amount'               => $stripe_gateway->format_amount( $donation_data['price'] ),
 					'currency'             => give_get_currency( $form_id ),
 					'payment_method_types' => [ 'card' ],
-					'statement_descriptor' => give_stripe_get_statement_descriptor(),
+                    'statement_descriptor_suffix' => give_stripe_get_statement_descriptor(),
```

Comment:
fyi, looks like both of these can be modified using the `give_stripe_create_intent_args` filter right now (if need be)

Reply:
Yup! True!